enterprise/server/api/api_server.go

```diff
@@ -62,10 +62,6 @@ func NewAPIServer(env environment.Env) *APIServer {
 }
 
 func (s *APIServer) checkPreconditions(ctx context.Context) (interfaces.UserInfo, error) {
```

Comment:
Do we still need this function if we've removed all the precondition checks?

At least it seems like we should rename it.

Reply:
Done (inlined the AuthenticatedUser checks instead of calling this func)